src/SharedAcceptanceTests.RequirePartitionKey/When_subscribers_handles_the_same_event.cs

```diff
@@ -0,0 +1,139 @@
+﻿namespace NServiceBus.AcceptanceTests.Outbox;
+
+using System;
+using System.Threading.Tasks;
+using AcceptanceTesting;
+using AcceptanceTesting.Customization;
+using AcceptanceTests;
+using EndpointTemplates;
+using Features;
+using NServiceBus.AcceptanceTesting.Support;
+using NUnit.Framework;
+
+public class When_subscribers_handles_the_same_event : NServiceBusAcceptanceTest
+{
+    [Test]
+    public async Task Should_be_processed_by_all_subscribers_using_default_synthetic_key()
+    {
+        Requires.OutboxPersistence();
+
+        var runSettings = new RunSettings();
+        runSettings.DoNotRegisterDefaultPartitionKeyProvider();
```

Comment:
We changed this but I've seen that this test is [included in Core](https://github.com/Particular/NServiceBus/blob/4cce8c188f4efd5eea3358ca8adc740cf9a9d4b7/src/NServiceBus.AcceptanceTests/Outbox/When_subscribers_handles_the_same_event.cs#L15), with the intention of it being run from there as opposed to having it included in each persistence. So may need to look into this further.